Utilities/bin_patch.cpp

```diff
@@ -562,43 +567,45 @@ static std::basic_string<u32> apply_modification(const patch_engine::patch_info&
 				// Revert if allocated map before failure
 				if (alloc_map->flags & (1ull << 62))
 				{
-					vm::unmap(vm::any, alloc_map->addr);
+					// Reset alloc_map before unmap (read to rvalue before resetting ptr)
+					vm::unmap(+alloc_map->addr + (alloc_map.reset(), 0), true);
```

Comment:
Uhh, there is no guarantee that the left part of `a + b` expression is evaluated first

Reply:
Fixed.